projects/Unit/TestConcurrentAccessWithSharedConnection.cs

```diff
@@ -150,20 +93,9 @@ public void TestConcurrentChannelOpenCloseLoop()
             }, 50);
         }
 
-        internal void TestConcurrentChannelOpenAndPublishingWithBodyOfSize(int length)
+        internal void TestConcurrentChannelOpenAndPublishingWithBodyOfSize(int length, int iterations = 30)
         {
-            TestConcurrentChannelOpenAndPublishingWithBodyOfSize(length, 30);
-        }
-
-        internal void TestConcurrentChannelOpenAndPublishingWithBodyOfSize(int length, int iterations)
-        {
-            string s = "payload";
-            if (length > s.Length)
-            {
-                s.PadRight(length);
```

Comment:
This didn't work as a string is immutable and the returned new string was unused. So it always sent "payload" for all length inputs. => I removed most of the different cases (when I changed it to send the actual size it ran into timeouts due to the huge amount of data it tried to send.)